src/test/resources/samples/json_merge/merged.jsonl

```diff
@@ -1,12 +1,20 @@
 {
   "key1": [
-    "value1",
-    "value3",
-    "value2"
+    "key1-value1",
+    "key1-value3",
+    "key1-value2"
   ],
   "key2": [
-    "value1",
-    "value3",
-    "value2"
-  ]
+    "key2-value1",
+    "key2-value3",
+    "key2-value2"
+  ],
+  "key3": [
+    "key3-value1"
+  ],
+  "key4": [
+    "key4-value1"
+  ],
+  "impossible_key1":"impossible_key1_value1",
+  "impossible_key2":{"key1":"value1"}
```

Comment:
Just in case, If incoming contains an invalid JSON field, we will just add it to the merged result. More importantly, the merging logic will not crash.